src/baldr/graphtile.cc

```diff
@@ -429,6 +429,10 @@ GraphTile::FileSuffix(const GraphId& graphid, const std::string& fname_suffix, b
 
   // figure out how many digits in tile-id
   const auto max_id = level.tiles.ncolumns() * level.tiles.nrows() - 1;
+  if (graphid.tileid() > max_id) {
```

Comment:
if we dont do this, `ind` below can wrap around 0 and cause all kinds of problems